en/security.md

```diff
@@ -96,6 +96,7 @@ class SessionController extends Controller
                 // The password is valid
             }
         } else {
+            // Protection against timing attacks
```

Comment:
Can we remove this from here? I want to be able to hide this whole code block from being translated. This is why I explain it further down.

Reply:
ok, removed